composer.json

```diff
@@ -79,7 +79,7 @@
         "doctrine/orm":                 "~2.2",
         "symfony/property-access":      "~2.3",
         "symfony/validator":            "~2.1",
-        "symfony/twig-bundle":          "~2.1"
+        "symfony/twig-bundle":          "~2.3"
```

Comment:
Don't think twig-bundle constraint should be updated.

Add a conflict about `twig/twig` package instead.